packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { ActionSheet, Text, TextField, TextFieldWithSteppers, ToggleSwitch } from '@deriv-com/quill-ui';
+import { Localize, localize } from '@deriv/translations';
+import RiskManagementInfoModal from '../RiskManagementInfoModal';
+import DealCancellationRemainingTime from '../DealCancellationRemainingTime/deal-cancellation-remaining-time';
+import { observer } from '@deriv/stores';
+import useContractDetails from 'AppV2/Hooks/useContractDetails';
+import { CONTRACT_TYPES, isAccumulatorContract, isValidToCancel } from '@deriv/shared';
+
+type RiskManagementItemProps = {
+    label: React.ReactNode;
+    modal_body_content: React.ReactNode;
+    is_deal_cancellation?: boolean;
+    value?: number | null;
+    type?: string;
+};
+
+const RiskManagementItem = observer(
+    ({ label, modal_body_content, is_deal_cancellation = false, value, type }: RiskManagementItemProps) => {
+        const [toggle, setToggle] = React.useState(Boolean(value));
+        const [isSheetOpen, setIsSheetOpen] = React.useState(false);
+        const [isEnabled, setIsEnabled] = React.useState(false);
+        const [stepperValue, setStepperValue] = React.useState(0);
+        const { contract_info, contract } = useContractDetails();
+        const { contract_type, currency } = contract_info;
+        const { validation_errors, updateLimitOrder, clearContractUpdateConfigValues } = contract;
+        const is_valid_to_cancel = isValidToCancel(contract_info);
+        const is_accumulator = isAccumulatorContract(contract_type);
+
+        React.useEffect(() => {
+            if (value) {
+                setStepperValue(value);
+                setToggle(Boolean(value));
+            }
+            return () => clearContractUpdateConfigValues();
+        }, [clearContractUpdateConfigValues, value]);
+
+        const isDealCancellation = is_valid_to_cancel;
+        const finalValue = Math.abs(value as number);
+
+        const errorKey = `contract_update_${type}` as 'contract_update_stop_loss' | 'contract_update_take_profit';
+        const errorMessage = validation_errors[errorKey]?.length > 0 ? validation_errors[errorKey][0] : '';
+
+        const getMessageForMultiplier = (is_valid_to_cancel: boolean, is_deal_cancellation: boolean) =>
+            is_valid_to_cancel && !is_deal_cancellation ? (
+                <Localize i18n_default_text='Take profit and stop loss are unavailable while deal cancellation is enabled.' />
+            ) : null;
+
+        const info_message = {
+            [CONTRACT_TYPES.ACCUMULATOR]: (
+                <Localize i18n_default_text='Take profit can’t be adjusted for ongoing accumulator contracts.' />
+            ),
+            [CONTRACT_TYPES.MULTIPLIER.UP]: getMessageForMultiplier(is_valid_to_cancel, is_deal_cancellation),
+            [CONTRACT_TYPES.MULTIPLIER.DOWN]: getMessageForMultiplier(is_valid_to_cancel, is_deal_cancellation),
```

Comment:
`getMessageForMultiplier(is_valid_to_cancel, is_deal_cancellation)` will return same value for both `CONTRACT_TYPES.MULTIPLIER.UP` and `CONTRACT_TYPES.MULTIPLIER.DOWN`

Reply:
If this function is used only in this place, maybe we can create a constant?
```
    const nessageForMultiplier = 
            is_valid_to_cancel && !is_deal_cancellation ? (
                <Localize i18n_default_text='Take profit and stop loss are unavailable while deal cancellation is enabled.' />
            ) : null;
```